docs/figure_file_format.rst

```diff
@@ -0,0 +1,223 @@
+
+OMERO.figure files
+==================
+
+OMERO.figure files are JSON data, stored in OMERO as OriginalFiles linked to FileAnnotations.
+
+File Annotations
+----------------
+
+The FileAnnotations are identified as OMERO.figure files using a namespace of ```omero.web.figure.json```.
+
+The description of the FileAnnotation can be used to specify an Image to show as a thumbnail
+in the File > Open listing of files::
+
+    {
+        "imageId": 123,
+        // only needed if image comes from a different server
+        "baseUrl": "demo.openmicroscopy.org",
+    }
+
+In general, OMERO.figure FileAnnotations are *not* linked to any Images because this would
+display them in the client UI alongside a delete button, allowing the user to accidentally delete
+the file, particularly since they may not recognise the file.
+
+JSON format
+-----------
+
+The JSON data format consists of a single 'Figure' object, with a list of 'panels'.
```

Comment:
The string 'Figure' is never used though, right? Perhaps remove it from the quotes?